.config-schema.json

```diff
@@ -32,6 +32,15 @@
             "minimum": 0,
             "default": 1000
         },
+        "enableExperimentals": {
+            "description": "a list of experimental features to enable",
+            "type": "array",
+            "items": {
+                "type": "string",
+                "enum": ["openfga.v1.list-objects-optimized"]
```

Comment:
Do we need the `openfga.v1` prefix for the flag?

Reply:
The reason I added it is to distinguish a feature specific to a version of the API (hence the server package version prefix). The thinking is that experimental features may be long lived and specific to one version of an API.

Reply:
Since this flag is planned to be so short-lived, I've renamed this experimental flag to simply `list-objects-optimized`

Reply:
I still see the old flag